### ISSUE TITLE
Support Erlang on Linux

### ISSUE THREAD

**Issue by lpil** [tsloughter/beamup#57]
```
[root@f5d5b424658c /]# beamup install erlang OTP-25.3.2.8
Error: install command not supported yet for language Erlang on "x86_64" "linux"

Location:
    src/main.rs:507:28
```

Where would the distribution be downloaded from? I couldn't find one to use.

edit: OK I've been looking around and the only Linux builds I can find are those in hexpm/bob, but they're Ubuntu only. Where do we get Erlang builds from? I've asked here: https://github.com/erlef/otp_builds/issues/39

**Comment by tsloughter** [comment]
Yes, I'm waiting on them o be in otp_builds but technically could with not too much work support switching to grabbing from bob when an install for Linux is requested.
